packages-next/admin-ui/src/components/Navigation.tsx

```diff
@@ -64,6 +71,21 @@ const NavItem: FunctionComponent<NavItemProps> = ({ href, children }) => {
   );
 };
 
+const SignoutButton: FunctionComponent = () => {
+  const [endSession, { loading, data }] = useMutation(END_SESSION);
+  useEffect(() => {
+    if (data?.endSession) {
+      top.location.reload();
```

Comment:
Why top instead of just window? (non-blocking comment just asking because I feel like a future person would go "I should leave it using top bc that was probably intentional")

Also, I'd probs await endSession() and do the reload after that bc I feel like it's easier to reason about but also it doesn't matter so do whatever you feel like